code/datums/diseases/transformation.dm

```diff
@@ -334,108 +334,6 @@
 				var/obj/item/I = affected_mob.get_active_held_item()
 				affected_mob.dropItemToGround(I)
 
-/datum/disease/transformation/felinid
```

Comment:
This is actually an old felinid disease not related to the species

Reply:
Plus if this is being removed, there is a string file that needs to be removed as well.
"strings/owo_talk.json"

Reply:
this does turn a player into a felinid species, i can make it just make you grow cat ears/tail instead

Reply:
Trashing the whole thing and making it an advanced symptom later might be best, especially with the words that owospeak gives you.

Reply:
Yeah, went ahead and removed it.